### ISSUE TITLE
`util.parseArgs()`: Uncaught TypeError: Cannot read properties of undefined (reading 'execArgv')

### ISSUE THREAD

**Issue by jcbhmr** [denoland/deno#22363]
Version: Deno 1.40.4

```sh
$ deno
Deno 1.40.4
exit using ctrl+d, ctrl+c, or close()
REPL is running with all permissions allowed.
To specify permissions, run `deno repl` with allow flags.
> let m = await import("node:util")
undefined
> m.parseArgs({ options: {}, allowPositionals: true })
Uncaught TypeError: Cannot read properties of undefined (reading 'execArgv')
    at getMainArgs (ext:deno_node/internal/util/parse_args/parse_args.js:53:28)
    at Module.parseArgs (ext:deno_node/internal/util/parse_args/parse_args.js:162:48)
    at <anonymous>:1:24
```

**Comment by jcbhmr** [comment]
looks like this:
https://github.com/denoland/deno/blob/24bdc1de33494bc1619bfebea826ab08ffb74a01/ext/node/polyfills/internal/util/parse_args/parse_args.js#L53

**Comment by javihernant** [comment]
I'd like to work on this!

**Comment by lucacasonato** [comment]
Fixes #22405
